python/tvm/topi/broadcast.py

```diff
@@ -22,7 +22,7 @@
 def broadcast_to(data, shape):
     """Broadcast the src to the target shape
 
-    We follow the numpy broadcasting rule.
+    We follows the numpy broadcasting rule.
```

Comment:
follow seeemed better to me if we say "we"...

Reply:
yeah this is actually a typo I fixed yesterday, but I reverted the commit that fixed this typo (to remove the change I made to `broadcast.h`, `broadcast.py` inside topi)